sdks/python/apache_beam/ml/transforms/tft.py

```diff
@@ -363,6 +363,42 @@ def apply_transform(
     return output
 
 
+@register_input_dtype(float)
+class ApplyBucketsWithInterpolation(TFTOperation):
+  def __init__(
+      self,
+      columns: List[str],
+      bucket_boundaries: Iterable[Union[int, float]],
+      name: Optional[str] = None):
+    """ Interpolates values within the provided buckest and then normalizes to
+    [0, 1].
+    
+    Input values are bucketized based on the provided boundaries such that the
+    input is mapped to a positive index i for which bucket_boundaries[i-1] <=
```

Comment:
do we need some escaping for code-snippets with backticks or something similar to make it look nicer in pydoc?

Reply:
We can try that out, added some backticks